cmd/main_test.go

```diff
@@ -125,8 +125,8 @@ var staticTests = []*cliTest{
 	},
 	{
 		name: "synthesize-only",
-		args: "--resource-input-file ../pkg/collector/data/json/Example1.json" +
-			" --synthesis-dump-dir examples/output/synthesis --disjoint-hint backend,frontend --disjoint-hint frontend,backend",
+		args: "--resource-input-file ../pkg/collector/data/json/Example1.json --synthesis-dump-dir examples/output/synthesis" +
+			" --disjoint-hint backend,frontend --disjoint-hint frontend,backend --create-policy-dns-server",
```

Comment:
test with value false for this flag?

Reply:
we do not check here ER, its just a change to see that the exec know this flag, changing it to false